apps/common-app/src/examples/AnimatedKeyboardExample.tsx

```diff
@@ -1,88 +1,168 @@
+import React from 'react';
 import Animated, {
-  KeyboardState,
   useAnimatedKeyboard,
   useAnimatedStyle,
+  useAnimatedProps,
 } from 'react-native-reanimated';
 import {
-  Button,
   Keyboard,
   ScrollView,
   StyleSheet,
   TextInput,
+  TouchableOpacity,
   View,
+  Text,
 } from 'react-native';
 
-import React from 'react';
+Animated.addWhitelistedNativeProps({ text: true });
+const AnimatedTextInput = Animated.createAnimatedComponent(TextInput);
 
-const BOX_SIZE = 50;
+const KeyboardStateToName = {
+  0: 'UNKNOWN',
+  1: 'OPENING',
+  2: 'OPEN',
+  3: 'CLOSING',
+  4: 'CLOSED',
+};
```

Comment:
Let's use `enum` here and probably change name to `KeyboardState`, i.e.:
```suggestion
enum KeyboardState = {
  0: 'UNKNOWN',
  1: 'OPENING',
  2: 'OPEN',
  3: 'CLOSING',
  4: 'CLOSED',
};
```